### DISCUSSION TITLE
Tutorial you didn't know you needed: make your data sing

### DISCUSSION THREAD

**Discussion by rlapere** [pangeo-data/escience-2022#29] [Show and tell]
Here is a short tutorial on how to convert a time series into a music file: https://github.com/rlapere/pymusic/blob/main/extract_music_tas.ipynb

The (very simple) example is based on surface temperature evolution in the Arctic and creates a melody in C minor. The MIDI file then needs to be opened with a dedicated software (e.g. GarageBand).

Have fun!

Thanks @MichaelSchulzMETNO for the inspiration!

**Comment by fairyeszhuo** [comment]
I create a volcanic eruption song, so cool, thanks a lot @rlapere

**Comment by theodorekhadir** [comment]
I love it, thanks for sharing :)
